var/spack/repos/builtin/packages/cdo/package.py

```diff
@@ -20,6 +20,11 @@ class Cdo(AutotoolsPackage):
 
     maintainers = ["skosukhin", "Try2Code"]
 
+    version(
+        "2.1.0",
+        sha256="b871346c944b05566ab21893827c74616575deaad0b20eacb472b80b1fa528cc",
+        url="https://code.mpimet.mpg.de/attachments/download/27481/cdo-2.0.6.tar.gz",
```

Comment:
Shouldn't this be `https://code.mpimet.mpg.de/attachments/download/27481/cdo-2.1.0.tar.gz`?

Reply:
sorry, my bad. I just fixed it